orb/core/connection_types/sql/postgres/statements/select.py

```diff
@@ -37,12 +37,33 @@ def __call__(self, model, context, fields=None):
         sql_columns = defaultdict(list)
         sql_joins = []
 
+        # pre-process columns for expansion based on shortcuts
+        cleaned_columns = []
+        for column in columns:
+            if column.shortcut() and not issubclass(model, orb.View):
```

Comment:
adding shortcut columns to being auto-expanded, so if you ask for a column that is a shortcut, you get it on initial selection vs. having to do a selection after the fact for that column:

```
class A(orb.Table):
    b = orb.ReferenceColumn('B')
    value = orb.StringColumn(shortcut='b.c.d')

class B(orb.Table):
    c = orb.ReferenceColumn('C')

class C(orb.Table):
     d = orb.StringColumn()
```

doing:

```
a = A(1)
```
will now auto-expand through b and c to get the d column value.